tests/widgets/test_menu.py

```diff
@@ -277,3 +298,87 @@ def test_data_choosers(asserter, qtbot, ij, gui_widget_chooser):
 
     # Assert that the data is now in napari
     asserter(lambda: 2 == len(button_to.viewer.layers))
+
+
+def test_settings_no_change(gui_widget: NapariImageJMenu):
+    """Ensure that no changes are made when there is no change from the defaults"""
+    button: SettingsButton = gui_widget.settings_button
+
+    # First record the old settings
+    old_yaml = [(k, v.get()) for k, v in settings.items()]
+
+    # Then update the settings, but select all defaults
```

Comment:
Maybe add a note that the above fixture is going to handle the settings selection for us? Otherwise the test here seems kinda pointless and magical...

Reply:
Added in 0de699e